### PR TITLE
Add Elastic Agent unenrollment issue

### PR DESCRIPTION
**Pull request by dedemorton**
Adds known issue item (related to https://github.com/elastic/fleet-server/issues/380) to the release notes.

### CODE REVIEW

**Comment by dedemorton** [comment]
@gchaps We are providing Fleet + Agent release notes in the Fleet docs, but I wonder if this should also be in the Kibana release notes, too, since the problem surfaces in Kibana? For now, we are duplicating the Kibana issues in the Fleet/Agent release notes to provide users with a full view of Fleet-related info.

File: docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc [hunk 1]
```diff
@@ -67,6 +72,24 @@ of {agent}s in {kib}.
...
+[[known-issue-25773]]
+.{agent}s unenrolling from a self-managed {fleet-server} hang at "Updating" and API keys are not invalidated
+[%collapsible]
+====
+
+*Details* 
+
+In {kib}, when you unenroll an {agent} from a self-managed {fleet-server},
+the status may hang at "Updating". This problem only occurs with {agent}s
+enrolled with a service token.
```
Comment (ruflin) [inline docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc:86]: This problem only occurs with Elastic Agents connecting to a `fleet-server` started with a service token.

**Comment by gchaps** [comment]
@dedemorton The 7.13 Kibana RN already list one known issue related to Fleet, so it seems natural to add the additional ones that relate to Kibana.

File: docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc (hunk 1 above)
Comment (dedemorton) [inline docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc:86]:
```suggestion
the status may hang at "Updating". This problem only occurs with {agent}s
connecting to a {fleet-server} started with a service token.
```

Comment (dedemorton) [reply docs/en/ingest-management/release-notes/release-notes-7.13.asciidoc]: Fixed. I dropped some detail there. :-)

**Review by ruflin — APPROVED** [review]

**Pull request merged**